apps/pie-microsite/src/assets/styles/base/_layout.scss

```diff
@@ -4,16 +4,10 @@
     min-height: 100%;
     display: grid;
     grid-template: 64px 1fr max-content / 260px 1fr;
-    grid-template-areas:
-            'header header'
-            'nav main'
-            'nav footer';
+    grid-template-areas: 'header header' 'nav main' 'nav footer';
```

Comment:
Why has this changed back?

Reply:
linting said it had to be! how did you avoid it?

Reply:
Was this a whilst pre-commiting? If so can you delete node modules and re-install. I added a fix to our config which should have pulled through from version 3.2.0 https://github.com/justeat/stylelint-config-fozzie/releases/tag/v3.2.0

Reply:
resolved the issue together offline - closing this thread :)